jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -103,15 +103,12 @@ bool ci_lazy_methodst::operator()(
 
   // Add any extra entry points specified; we should elaborate these in the
   // same way as the main function.
-  std::vector<irep_idt> extra_entry_points;
   for(const auto &extra_function_generator : lazy_methods_extra_entry_points)
   {
-    const auto &extra_methods = extra_function_generator(symbol_table);
-    extra_entry_points.insert(
-      extra_entry_points.end(), extra_methods.begin(), extra_methods.end());
+    const std::vector<irep_idt> &extra_methods =
```

Comment:
Nitpick: not keen on the const-ref here, even though lifetime extension means it's not as dangerous as it looks